content/hi/docs/concepts/overview/working-with-objects/owners-dependents.md

```diff
@@ -0,0 +1,88 @@
+---
+title: मालिक और आश्रित
+content_type: concept
+weight: 90
+---
+
+<!-- overview -->
+
+कुबेरनेट्स में, कुछ {{< glossary_tooltip text="objects" term_id="object" >}} हैं
+*owners* अन्य वस्तुओं का. उदाहरण के लिए,
+{{<glossary_tooltip text="ReplicaSet" term_id="replica-set">}} है owner
+पॉड्स के एक सेट का। ये स्वामित्व वाली वस्तुएँ हैं *dependents* उनके owner.
+
+स्वामित्व से भिन्न है [लेबल और चयनकर्ता](/docs/concepts/overview/working-with-objects/labels/)
+वह तंत्र जिसका उपयोग कुछ संसाधन भी करते हैं। उदाहरण के लिए, एक ऐसी सेवा पर विचार करें
+बनाता है `EndpointSlice` वस्तुएं. सेवा उपयोग करती है {{<glossary_tooltip text="labels" term_id="label">}} नियंत्रण विमान को अनुमति देने के लिए
+निर्धारित करें कि कौन सा `EndpointSlice` उस सेवा के लिए वस्तुओं का उपयोग किया जाता है। इसके अलावा
+प्रत्येक लेबल पर `EndpointSlice` जिसे किसी सेवा की ओर से प्रबंधित किया जाता है
+एक मालिक संदर्भ. स्वामी संदर्भ कुबेरनेट्स के विभिन्न हिस्सों से बचने में मदद करते हैं
+उन वस्तुओं में हस्तक्षेप करना जिन पर उनका नियंत्रण नहीं है।
+
+## वस्तु विशिष्टताओं में स्वामी संदर्भ
+
+आश्रित वस्तुओं में एक है `metadata.ownerReferences` वह क्षेत्र जो उनका संदर्भ देता है
+स्वामी वस्तु. एक वैध स्वामी संदर्भ में वस्तु का नाम और शामिल होता है {{<glossary_tooltip text="UID" term_id="uid">}} 
+उसी के भीतर {{<glossary_tooltip text="namespace" term_id="namespace">}} आश्रित वस्तु के रूप में। कुबेरनेट्स का मान निर्धारित करता है
+यह फ़ील्ड स्वचालित रूप से उन वस्तुओं के लिए है जो अन्य वस्तुओं पर निर्भर हैं
+रेप्लिकासेट्स, डेमॉनसेट्स, परिनियोजन, नौकरियां और क्रोनजॉब्स, और रेप्लिकेशनकंट्रोलर।

```

Comment:
ReplicaSets, DaemonSets, Deployments, Jobs and CronJobs, and ReplicationControllers.
I think its better to be in English rather than in hindi. WDYT?